### DISCUSSION TITLE
Modify move directions for automapper

### DISCUSSION THREAD

**Discussion by buhtz** [scandum/tintin#174] [General]
I would like to use the automapper but having a german MUD where `east` is `ost`.

But the automapper only reactions on `e` not `o`. Can I configure that?

I don't want to create a workaround like an alias in the MUD itself. I don't want to use `e` instead of `o`.

**Comment by scandum** [comment]
See `#help pathdir` on how to create custom directions.
```
#PATHDIR {o} {w} {2}
#PATHDIR {w} {o} {8}
``
Adding that to your command file should do the trick.

**Reply by buhtz** [reply]
northeast would be 3?
southeast would be 6?

```
#PATHDIR {no} {sw} {3}
#PATHDIR {so} {nw} {6}
```

Am I right?

**Comment by scandum** [comment]
Correct.
